docs/content/guides/developer/getting-started/configure-sui-client.mdx

```diff
@@ -0,0 +1,128 @@
+
+:::
+
+</TabItem>
+
+<TabItem label="Windows" value="windows">
+
+Run the Sui CLI with the command:
+
+```
+$ sui client
+```
+
+The CLI prompts you to create a new Sui address and setup a network environment configuration. By default, Sui stores this information in the `%USERPROFILE%\.sui\sui_config\client.yaml` file. You can store a `client.yaml` file in a different location, if preferred, and specify it's location with the `--client.config` flag. 
+
+Sui stores the key for the Sui address in a separate file, `%USERPROFILE/.sui/sui_config/sui.keystore`. Learn more about Sui addresses in [Create a Sui Address](/guides/developer/getting-started/get-address.mdx).
+
+:::caution
+
+If you already have a `client.yaml` file stored locally from a previous Sui installation, you will receive the `sui client --help` output in the console. You can delete the existing `%USERPROFILE%\.sui\sui_config\client.yaml` file if you'd like to start fresh, or you can continue using the existing configuration. 
+
+:::
+
+</TabItem>
+</Tabs>
+
+
+### Configuration options
+
+The first prompt will confirm if you'd like to create the `client.yaml` file, select `Y`:
+
+```
+Config file ["<PATH-TO-FILE>/client.yaml"] doesn't exist, do you want to connect to a Sui full node server [y/N]?
+```
+
+The second prompt configures the network server URL that you'd like to connect to:
+
+```
+Sui full node server URL (Defaults to Sui Testnet if not specified) :
+```
+
+If you specify nothing, the client uses Testnet by default. Using the Testnet is recommended for developers new to Sui that do not have a project already developed. 
+
+The options are:
+
+- `https://fullnode.testnet.sui.io:443`: The RPC URL for the Testnet network.
+
+- `https://fullnode.devnet.sui.io:443`: The RPC URL for the Devnet network. Devnet is not recommended for new Sui developers, as it is better suited for more advanced use cases and the network wipes data weekly as part of its update process.
+
+- `https://fullnode.mainnet.sui.io:443`: The RPC URL for the Mainnet network. Publishing applications to the Mainnet costs real SUI tokens. 
+
+- `http://0.0.0.0:9000`: The RPC URL for a Localnet network, if configured. [Learn how to setup a local network](/guides/developer/sui-101/local-network.mdx).
+
+<ImportContent source="data-wipe.mdx" mode="snippet" />
+
+The final prompt is to select an encryption scheme. Once you select a scheme, the client will generate a new Sui address. Press **0**, **1**, or **2** to select a key scheme:
+
+```
+Select key scheme to generate key pair (0 for ed25519, 1 for secp256k1, 2 for secp256r1):
+```
+
+<ImportContent source="crypto-schemes.mdx" mode="snippet" />
+
+[Learn more about Sui addresses, key generation, and recovery phrases](/guides/developer/getting-started/get-address.mdx).
```

Comment:
Again, i would do just Learn more as the hyperlink. I haven't looked at the output, but seems like there'd be a lot of blue.